### PR TITLE
feat: playbook parameter types

### PR DESCRIPTION
**Pull request by adityathebe**
resolves: https://github.com/flanksource/docs/issues/149

### CODE REVIEW

File: mission-control/docs/playbooks/concepts/playbook.md [hunk 1]
```diff
@@ -68,8 +68,71 @@ Playbook parameter defines a parameter that a playbook needs to run.
...
+| `list`      | Specify a custom list of values    | Dropdown     | `string`  | [`List Property`](#list-parameter-properties)           |
+| `people`    | Limits the value to people.        | Dropdown     | `string`  | [`People Property`](#people-parameter-properties)       |
+| `team`      | Limits the value to teams.         | Dropdown     | `string`  | -                                                       |
+| `text`      | Text input                         | Text Input   | `string`  | [`Text Property`](#text-parameter-properties)           |
+
+##### `check` parameter properties
+
+| Field    | Description                        | Schema   |
+| -------- | ---------------------------------- | -------- |
+| `filter` | Limit the checks to the given type | `string` |
```
Comment (moshloop) [inline mission-control/docs/playbooks/concepts/playbook.md:92]: Isn't this a CEL expression ?

Comment (adityathebe) [reply mission-control/docs/playbooks/concepts/playbook.md:92]: How so? Isn't this a just a WHERE clause on the PostgREST call?

File: mission-control/docs/playbooks/concepts/runners.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    - name: send heartbeat
+      exec:
+        env:
+          - name: HEARTBEAT_TOKEN
+            valueFrom:
+              secretKeyRef:
+                name: canary-checker-heartbeat
+                key: token
+        script: |
+          curl -H "Authorization: $HEARTBEAT_TOKEN" https://canarychecker.com/heartbeat/{{.params.check}}
```
Comment (moshloop) [inline mission-control/docs/playbooks/concepts/runners.md:73]: Can we change this to httpbin?

File: mission-control/src/components/Badges.jsx [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
```
Comment (moshloop) [inline mission-control/src/components/Badges.jsx:2]: the mission-control src/ folder can be gitignored as its copied during make

**Pull request merged**